mne/viz/evoked.py

```diff
@@ -2122,22 +2122,21 @@ def plot_compare_evokeds(evokeds, picks=None, gfp=False, colors=None,
     # and now for 3 "legends" ..
     # a head plot showing the sensors that are being plotted
     if show_sensors:
-        if not _check_ch_locs(one_evoked.info['chs']):
-            pos = _auto_topomap_coords(one_evoked.info, pos_picks,
-                                       ignore_overlap=True, to_sphere=True)
-        else:
+        _validate_type(show_sensors, (np.int, bool, str, type(None)),
+                       "show_sensors", "numeric, str, None or bool")
+        if not _check_ch_locs(np.array(one_evoked.info['chs'])[pos_picks]):
             warn("Cannot find channel coordinates in the supplied Evokeds. "
                  "Not showing channel locations.")
-
-        if show_sensors is True:
-            ymin, ymax = np.abs(ax.get_ylim())
-            show_sensors = "lower right" if ymin > ymax else "upper right"
         else:
+            if show_sensors is True:
```

Comment:
shouldn't this be an `elif`?

Reply:
I was suspicious about this part when I edited @jona-sassenhagen but no tests stopped me. Is it easy to add a tiny test for this?

Reply:
no, why an elif?

Reply:
oh wait, `L2135` is nested inside the `else` ? Then it's good ...

Reply:
Yeah I think this is what originally broke it.

2131 is just to get a default for the location.